src/components/ColorPicker/BrandColorPicker.tsx

```diff
@@ -56,6 +56,7 @@ export const BrandColorPicker: FC<Props> = ({ palettes: defaultPalettes = [], cu
                         decorator={<IconSearch />}
                         placeholder="Search"
                         onChange={(value) => setQuery(value)}
+                        clearable
```

Comment:
This fixes `Search input field should have a X when it is filled to quickly remove search term`